mlflow/alembic/versions/451aebb31d03_add_metric_step.py

```diff
@@ -0,0 +1,35 @@
+"""add metric step
```

Comment:
Should be reviewed, adds metric step column

Reply:
What was the process (i.e., set of commands) by which you generated this?

Reply:
(Or was it really just run the one `alembic revision` command and then the rest by hand?)

Reply:
It was essentially running `alembic revision` & adding the rest by hand - I initially used `alembic revision --autogenerate`, which produced the `op.add_column('metrics', sa.Column('step', sa.BigInteger(), nullable=False, server_default='0'))` command, but had to add the `batch_op` stuff myself (alembic's autogeneration didn't include batch mode or the primary key change)